Aplib.Core/Intent/Tactics/FirstOfTactic.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using Aplib.Core.Belief.BeliefSets;
 using Aplib.Core.Intent.Actions;
+using System;
```

Comment:
System.Action clashes with Aplib.Core.Intent.Actions.Action, so I would like to prevent using System in Aplib.Core at all.
```suggestion
```